.circleci/config.yml

```diff
@@ -465,13 +465,16 @@ jobs:
           test_targets: "posixtest"
   test-core0:
     executor: bionic
+    environment:
+      EMTEST_SKIP_NODE_CANARY: "1"
```

Comment:
@sbc100 Is this what you had in mind? It seems a bit awkward to have to spread these skips everywhere that isn't node canary.

I think an alternative that could be better instead of using `requires_node_canary`, is to have the node canary test run set an environment variable and then have something like `skipIf(!os.environ.isNodeCanary)` in the python test.

Reply:
If you look at how `test-mac` and `test-windows` are currently setup the have opt out of all kinds of things:

e.g:

```
      EMTEST_LACKS_NATIVE_CLANG: "1"                                                
      EMTEST_SKIP_V8: "1"                                                           
      EMTEST_SKIP_EH: "1"                                                           
      EMTEST_SKIP_WASM64: "1"                                                       
      EMTEST_SKIP_SIMD: "1"                                                         
      EMTEST_SKIP_SCONS: "1"        
 ```

I guess we we could invert the logic for node canary.. but its its really any different the v8 tests?

Could the bionic executor set `EMTEST_SKIP_NODE_CANARY` avoiding the need to duplicate this in each place?  Then the node canary test runner could override this?

Reply:
It sounds like you are suggest `EMTEST_ENABLE_NODE_CANARY` as an opt in rather than `EMTEST_SKIP_NODE_CANARY` as an opt out?   

I guess maybe that does sounds reasonable in this case..

Reply:
Yeah, more as opt-in which seems to match how node canary would be used. If we're not doing the inverted logic for any other tests I'm fine with keeping it as skip though. Any preference?

Reply:
I'll leave it up to you.  I think node canary is rare enough that perhaps opt-in is fine in this case.

Reply:
I played a bit with converting it, but it felt too different to how current requires work. I'll leave as is.

Fine to land now?